src/app/pages/storage/VMware-snapshot/VMware-snapshot/VMware-snapshot-form.component.ts

```diff
@@ -257,7 +257,11 @@ export class VMwareSnapshotFormComponent {
         (error)=>{
           this.datastore.options.length = 0;
           parent.loader.close();
-          parent.dialogService.errorReport(error.error,error.reason, error.trace.formatted);
+          if (error.reason.includes('[ETIMEDOUT]')) {
```

Comment:
you might want to do if (error.reason && error.reason.includes('[ETIMEDOUT]')) just in case the reason isn't even there.